effekt/jvm/src/test/scala/effekt/core/VMTests.scala

```diff
@@ -357,8 +357,8 @@ class VMTests extends munit.FunSuite {
       poppedFrames = 8661,
       allocations = 0,
       closures = 0,
-      variableReads = 32437,
-      variableWrites = 13699,
+      variableReads = 23776,
+      variableWrites = 5039,
```

Comment:
We should instrument the refs at some point (can you open an issue?)

Reply:
Done, #916